neutrino.go

```diff
@@ -935,6 +964,33 @@ func NewChainService(cfg Config) (*ChainService, error) {
 	if err != nil {
 		return nil, err
 	}
+
+	sideload := &sideloader{}
+	if cfg.BlkHdrSideloader != nil {
+		sd, err := prepBlkHdrSideload(cfg.BlkHdrSideloader, hdrProcessor,
+			sideload)
+
+		if err != nil {
+			return nil, err
+		}
+
+		if sd != nil {
+			s.blkHdrSideloader = sideload
+		}
+	}
+
+	if cfg.CfHdrSideloader != nil {
```

Comment:
Perhaps we can move this logic into a function that handles reading the config and initializing the proper side loader?

Reply:
ok thanks